CvGameCoreDLL/CvPlot.cpp

```diff
@@ -6908,6 +6911,26 @@ int CvPlot::calculateYield(YieldTypes eYield, bool bDisplay) const
 	}
 
 	iYield = calculateNatureYield(eYield, ((ePlayer != NO_PLAYER) ? GET_PLAYER(ePlayer).getTeam() : NO_TEAM));
+	
+	// Leoreth + Merijn: Burj Khalifa effect

```

Comment:
Have you checked if the changed output is displayed in the city screen even without working the tile?

Reply:
It does.

If a tiles can be worked by multiple cities, it will show the yield of the city that "controls" that tile. (Even if it is not currently worked)